src/app/service/cityio.service.ts

```diff
@@ -0,0 +1,23 @@
+import { Injectable } from '@angular/core';
+import { HttpClient } from '@angular/common/http'
+import { interval } from 'rxjs';
+
+@Injectable({
+  providedIn: 'root'
+})
+export class CityioService {
+
+  baseUrl = "https://cityio.media.mit.edu/api/table/";
+  tableName = "virtual_table";
+  url = `${this.baseUrl}${this.tableName}`;
+
+  table_data: any = null; // can be accessed by other components, this will always be updated
```

Comment:
I'm initiating with null, so other components doesn't get `undefined` when trying to look into it.